corehq/apps/domain/forms.py

```diff
@@ -1266,6 +1267,13 @@ def __init__(self, domain, can_edit_eula, *args, **kwargs):
             ),
         )
 
+        if not self.user.is_staff:
```

Comment:
Is it possible for `self.user` to be `None`? If yes, this will throw `AttributeError`. If not, the `user` arg in the constructor could be changed to a required keyword-only argument:
```py
def __init__(self, domain, can_edit_eula, *args, user, **kwargs):
```